Setup.sh

```diff
@@ -0,0 +1,136 @@
+
+markStartOfBlock "Retrieve dependencies"
+    spatial package retrieve tools           schema_compiler-x86_64-win32               $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/tools/schema_compiler-x86_64-win32.zip
+    spatial package retrieve schema          standard_library                           $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/schema/standard_library.zip
+    spatial package retrieve worker_sdk      c-dynamic-x86-msvc_md-win32                $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/worker_sdk/c-dynamic-x86-msvc_md-win32.zip
+    spatial package retrieve worker_sdk      c-dynamic-x86_64-msvc_md-win32             $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-msvc_md-win32.zip
+    spatial package retrieve worker_sdk      c-dynamic-x86_64-gcc_libstdcpp-linux       $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-gcc_libstdcpp-linux.zip
+    spatial package retrieve worker_sdk      c-dynamic-x86_64-clang_libcpp-macos        $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-clang_libcpp-macos.zip
+    spatial package retrieve worker_sdk      c-static-fullylinked-arm-clang_libcpp-ios  $PINNED_CORE_SDK_VERSION       $CORE_SDK_DIR/worker_sdk/c-static-fullylinked-arm-clang_libcpp-ios.zip
+markEndOfBlock "Retrieve dependencies"
+
+markStartOfBlock "Unpack dependencies"
+    unzip -oq $CORE_SDK_DIR/worker_sdk/c-dynamic-x86-msvc_md-win32.zip                 -d $BINARIES_DIR/Win32/
+    unzip -oq $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-msvc_md-win32.zip              -d $BINARIES_DIR/Win64/
+    unzip -oq $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-gcc_libstdcpp-linux.zip        -d $BINARIES_DIR/Linux/
+    unzip -oq $CORE_SDK_DIR/worker_sdk/c-dynamic-x86_64-clang_libcpp-macos.zip         -d $BINARIES_DIR/Mac/
+    unzip -oq $CORE_SDK_DIR/worker_sdk/c-static-fullylinked-arm-clang_libcpp-ios.zip   -d $BINARIES_DIR/IOS/
+    unzip -oq $CORE_SDK_DIR/tools/schema_compiler-x86_64-win32.zip                     -d $BINARIES_DIR/Programs/
+    unzip -oq $CORE_SDK_DIR/schema/standard_library.zip                                -d $BINARIES_DIR/Programs/schema/
+
+    cp -R $BINARIES_DIR/Mac/include/ $WORKER_SDK_DIR
+markEndOfBlock "Unpack dependencies"
+
+markStartOfBlock "Copy standard library schema"
+    echo "Copying standard library schemas to $SCHEMA_STD_COPY_DIR"
+    cp -R $BINARIES_DIR/Programs/schema $SCHEMA_STD_COPY_DIR
+markEndOfBlock "Copy standard library schema"
+
+markStartOfBlock "Copy GDK schema"
+    rm -rf $SCHEMA_COPY_DIR   2>/dev/null
+    mkdir -p $SCHEMA_COPY_DIR >/dev/null 2>/dev/null
+
+    echo "Copying schemas to $SCHEMA_COPY_DIR."
+    cp -R $(dirname %0)/SpatialGDK/Extras/schema $SCHEMA_COPY_DIR
+markEndOfBlock "Copy GDK schema"
+
+markStartOfBlock "Build C# utilities"
+    msbuild /nologo /verbosity:minimal ./SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Mac/Improbable.Unreal.Scripts.sln /property:Configuration=Release
```

Comment:
This file seems to be missing: `SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Mac/Improbable.Unreal.Scripts.sln`.
It is most likely because `*.sln` is ignored by `.gitignore`. Please update your `.gitignore` by adding the second line:
`.gitignore`, line 82:
```
!SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Improbable.Unreal.Scripts.sln
!SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Mac/Improbable.Unreal.Scripts.sln
```
Then, commit the `.sln` file.

Reply:
Nice catch!